### ISSUE TITLE
Camouflage mode for public/safe viewing

### ISSUE THREAD

**Issue by switzersc** [codefordenver/sol-cavp#52]

**Comment by dviramontes** [comment]
@vallejoj im thinking this can be worked on very much independently of other _features_, meaning
since @zachwood and I cloned the site last night, we could technically start working on this feature
against the static assets.

Thoughts ?

**Comment by vallejoj** [comment]
Yeah totally!  I just wanted to get started on something and am missing a little direction, but if you are using Dropal to make an emergency button the best I figured is that you would have to add another tab to make it. I can not find any documentation on a button that is a link that is not just simply a tab button.

**Comment by vallejoj** [comment]
N/M just saw that they have a volunteer button, I just have zero clue how they did it. Not that I am too savvy on this either way.

**Comment by dkoloditch** [comment]
just a thought on details... i think jQuery could easily leveraged for this. maybe with a modal to quickly inform the user of the feature on first load, along with a simple and static page that overlays everything when activated. jsfiddle prototype: https://jsfiddle.net/asfwevahrg/wtg3ruhv/1/

**Comment by vallejoj** [comment]
Yeah that sounds great!!! I was checking what the world has to say about adding jquery to Drupal, and its doable. I know we are not there yet, but here is a link on how it can be done http://dannyenglander.com/blog/jquery-theming-tips-drupal-7.
